llvm/lib/Transforms/Scalar/IR2Vec-LOF/custom_loop_distribution/custom_loop_distribution.cpp

```diff
@@ -296,6 +297,7 @@ bool custom_loop_distribution::runOnFunction(Function &F) {
     MLRunner->setRequest(&request);
     MLRunner->setResponse(&response);
 
+    outfile.open("grpc_out.log", std::ios_base::app);
```

Comment:
Why is this needed? Please revert.

Reply:
Similarly to the pipes dumping the sequences generated while inference to this log file to compare with pipes and onnx.

Reply:
removed